src/docs/doxygen/GeosxConfig.hpp

```diff
@@ -72,11 +72,11 @@
 #define GEOSX_USE_PETSC
 
 /// Choice of global linear algebra interface (CMake option GEOSX_LA_INTERFACE)
-#define GEOSX_LA_INTERFACE Hypre
+#define GEOSX_LA_INTERFACE Trilinos
```

Comment:
you can build this on quartz to get this file back to the default.

Reply:
Done.